src/Service/BedrockRuntime/tests/Unit/Result/ConverseResponseTest.php

```diff
@@ -0,0 +1,310 @@
+<?php
+
+namespace AsyncAws\BedrockRuntime\Tests\Unit\Result;
+
+use AsyncAws\BedrockRuntime\Result\ConverseResponse;
+use AsyncAws\Core\Response;
+use AsyncAws\Core\Test\Http\SimpleMockedResponse;
+use AsyncAws\Core\Test\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpClient\MockHttpClient;
+
+class ConverseResponseTest extends TestCase
+{
+    public function testConverseResponse(): void
+    {
+        // see https://docs.aws.amazon.com/bedrock/latest/APIReference/API_Converse.html
+        $response = new SimpleMockedResponse('{
+   "metrics":{
+      "latencyMs":100
+   },
+   "output":{
+      "message":{
+         "content":[
+            {
+               "role":"user",
```

Comment:
this `"role": "user" should be a key in the Message object, not in the ContentBlock object (this is why the tests are failing)